bundle/bin/rke2-uninstall.sh

```diff
@@ -7,7 +7,11 @@ if [ ! $(id -u) -eq 0 ]; then
     exit 1
 fi
 
-: "${INSTALL_RKE2_ROOT:="/usr/local"}"
+if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ]; then
+    : "${INSTALL_RKE2_ROOT:="/usr"}"
+else
+    : "${INSTALL_RKE2_ROOT:="/usr/local"}"
+fi
```

Comment:
P.S. This will fail when uninstalling a tarball installation on a redhat-derived system.

Reply:
Should we perhaps just check for /usr/local/bin/rke2 as an indicator of a tarball-based install?

Reply:
Maybe we can update to that logic later on. For now I think this covers most cases (are people really installing RKE2 using tarball that often?)

Reply:
Do we want this change now or later? What's the cncensus?

Reply:
I am fine with changing this later

Reply:
I just don't want to lose track of it. Can we open a small gh issue to track? I guess set v1.19.5 milestone to fix this soon? Easy to fix later when we have some more time?
Let's just fix it later - but create gh issue to track.